### PR TITLE
Add images

### PR DESCRIPTION
**Pull request by Michael3Healy**
### Description

Added images to represent the codebytes given in examples 1 and 2 of CSS/Position/top. Closes issue #4075 

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@Michael3Healy You need to move the images `abs_pos_40px.png` and `rel_pos_100px.png` to the `media` folder in docs - `docs/media` file path. I would also suggest you rename the images to something like this (please feel free to rename them in another way, you can take a look at the existing images in the `media` folder to get a better idea):
`abs_pos_40px.png` => `css_top_position_absolute.png`
`rel_pos_100px.png` => `css_top_position_relative.png`

**Comment by Michael3Healy** [comment]
Thanks @ishg-153, I made the changes.

File: content/css/concepts/position/terms/top/top.md [hunk 1]
```diff
@@ -43,6 +43,10 @@ Set the position of `.box` element `40px` from the top of the nearest relative p
 }
 ```
 
+![CSS Top Attribute Absolute](images/abs_pos_40px.png)
```
Comment (ishg-153) [inline content/css/concepts/position/terms/top/top.md:46]:
```suggestion
![Shows use CSS top attribute with element position set to absolute](images/abs_pos_40px.png)
```
Make the `alt-text` (text inside square brackets) more descriptive, like so. Also, add the updated image link (inside round brackets) in this format `https://raw.githubusercontent.com/Codecademy/docs/main/media/` followed by the image name.

File: content/css/concepts/position/terms/top/top.md [hunk 2]
```diff
@@ -56,3 +60,7 @@ Set the position of `.box` element `100px` from the elements top edge.
   top: 100px;
 }
 ```
+
+![CSS Top Attribute Relative](images/rel_pos_100px.png)
```
Comment (ishg-153) [inline content/css/concepts/position/terms/top/top.md:64]: In a similar fashion mentioned in the suggestion above, update the `alt-text` and the image link.

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@Michael3Healy I have suggested two more changes. Please address them and let me know once you are done.

File: content/css/concepts/position/terms/top/top.md (hunk 2 above)
Comment (Michael3Healy) [reply content/css/concepts/position/terms/top/top.md]: Done.

**Review by ishg-153 — APPROVED** [review]
@Michael3Healy Passing this on for a second review!

**Review by SSwiniarski — APPROVED** [review]
@Michael3Healy One small change and it looks good.

**Comment by Michael3Healy** [comment]
@SSwiniarski Why aren't the images displaying correctly on the Codecademy site?

**Comment by SSwiniarski** [comment]
@Michael3Healy I just checked and the links are wrong. They're named differently in the link and the media folder. Do you want to submit a PR to correct the links?

**Comment by Michael3Healy** [comment]
@SSwiniarski Done - #4166

**Comment by Michael3Healy** [comment]
@SSwiniarski Done - #4204

**Pull request merged**